layers/+lang/haskell/README.org

```diff
@@ -283,9 +283,12 @@ Refactor commands are prefixed by ~SPC m r~:
 |-------------+---------------------------------------------------|
 | ~SPC m r b~ | apply all HLint suggestions in the current buffer |
 | ~SPC m r r~ | apply the HLint suggestion under the cursor       |
+| ~SPC m r s~ | list all Intero suggestions                       |
 
 Only some of the HLint suggestions can be applied.
 
+To apply the intero suggestions, press `C-c C-c` when the window is open, which is also shown in the window that appears.
```

Comment:
Hey,

The last thing, could you please `fill-paragraph` here? With emacs editing style it's `M-q` and should be the same for vim editing style, but I am not 100% sure. Thanks.